web-api/nishiki-web-api.yaml

```diff
@@ -445,6 +445,42 @@ paths:
                   groupId:
                     type: string
                     example: d4825cf5-0b1b-dce3-1aa1-ed9adfd6b0bd
+  /groups/?Action=joinToGroup:
```

Comment:
I created a join to a group API as a put method.
The joining user won't know the target group ID, so I decided to not use the {groupId}. This API returns a Group ID that a user joined using this API.

Reply:
@hitohata 
Thank you for your description. Let me confirm the behavior. When this endpoint is called, it just returns groupId based on invitationLink, so afterward PUT `/groups/{groupId}/users/{userId}` must be called to add the user to the group. Is this understanding correct?

Reply:
@taniguchiiqqq 

Thank you for your comment.

>so afterward PUT /groups/{groupId}/users/{userId} must be called to add the user to the group. Is this understanding correct?

No, this endpoint will add a user. Therefore, the client (front) doesn't need to more action.

Reply:
@hitohata 
I got it. Thanks. Then do you think `PUT /groups/{groupId}/users/{userId}` is neccesarry? When a user creates a group, a user can be added to the group just by calling `POST /groups/{groupId}` as the join to a group endpoint does.

Or both when a user creates a group and when a user joins a group, `PUT /groups/{groupId}/users/{userId}` should be called after getting groupId.

Reply:
@taniguchiiqqq 
It made sense. So, I need to remove the  `PUT /group/{groupId}/users/{userId}` API.

Reply:
@hitohata 
> I need to remove the PUT /group/{groupId}/users/{userId} API.

Yes, I agree with this.

Reply:
I deleted it.